roles/openshift_health_checker/library/aos_version.py

```diff
@@ -78,7 +82,7 @@ def bail(error):  # pylint: disable=missing-docstring
         for name in not_found:
             msg += '  %s\n' % name
         bail(msg + 'Please check your subscriptions and enabled repositories.')
-    if multi_found:
+    if multi_found and rpm_prefix == 'atomic-openshift':
```

Comment:
It's weird to rely on the "rpm prefix" for this. As we've seen, there are multiple prefixes that mean "enterprise", depending on the version.

If we want to explicitly disable this check for Origin, I'd suggest we use a mechanism like we use for containerized / non-containerized.